### PR TITLE
Update free-programming-books-hi.md

### PR DESCRIPTION
**Pull request by sharma62**
> ### Core PHP Hindi/Urdu

## What does this PR do?
PHP Hindi/Urdu course | NA | Online video HD | Improve repo add the course.

## For resources### Php
* [Core PHP Tutorial \| Hindi](https://youtube.com/playlist?list=PL0b6OzIxLPbyrzCMJOFzLnf_-_5E_dkzs&si=tiekSh6nKxtn0DmC)) - Yahoo baba

### Description
 About the course Core PHP as well as dive to depth concepts so that you visualize your concept.
### Why is this valuable (or not)?
Because I learned the entire `core php ` through it. believe me, it is good and the instructor is also polite. 

### How do we know it's really free?
Because it's on YouTube.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Don't invent titles! Use the title given by the resource.  Says Hind/Urdu.

**Comment by sharma62** [comment]
> Don't invent titles! Use the title given by the resource. Says Hind/Urdu.

@eshellman I'll update the title please review it, master.

File: books/free-programming-books-hi.md [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
 ### Php
 
+* [Core PHP Tutorial \| Hindi](https://youtube.com/playlist?list=PL0b6OzIxLPbyrzCMJOFzLnf_-_5E_dkzs&si=tiekSh6nKxtn0DmC) - Yahoo baba
```
Comment (eshellman) [inline books/free-programming-books-hi.md:52]:
```suggestion
* [Core PHP Tutorial \| Hindi](https://youtube.com/playlist?list=PL0b6OzIxLPbyrzCMJOFzLnf_-_5E_dkzs) - Yahoo baba
```

**Comment by eshellman** [comment]
something's wrong with the branch

**Pull request closed without merging**